tests/fixtures/client.py

```diff
@@ -59,6 +60,7 @@ async def connect(
         self.server = self._client.server
         self.app = self.server.app
         self.db = self.app.get("db", None)
+        self.postgres = self.app.get("postgres", None)
```

Comment:
Can be simplified to ``self.db = self.app.get("db")``. The default value for ``default`` in `get()` is `None`.

Please rename `postgres` to `pg`. Trying to standardize on this.